docs/install/offline.md

```diff
@@ -234,6 +235,81 @@ server, as demonstrated in the example below:
 With these steps, you'll have the Coder documentation hosted on your server and
 accessible for your team to use.
 
+## Using Registry Modules Offline
+
+Coder Registry modules from [registry.coder.com](https://registry.coder.com) provide IDEs, tools, and integrations for workspaces. In air-gapped environments, you have several options:
+
+### Option 1: Git Repository (Recommended)
+
+Mirror registry modules in an internal Git repository:
+
+1. **Download modules** from [registry.coder.com](https://registry.coder.com) while connected to the internet
+2. **Store modules** in your internal Git repository:
+
+   ```text
+   internal-registry/
+   ├── modules/
+   │   ├── code-server/
+   │   ├── cursor/
+   │   └── vscode-web/
+   └── templates/
+   ```
+
+3. **Reference modules** using Git source addresses:
+
+   ```tf
+   module "code_server" {
+     source   = "git::https://your-internal-git.com/coder-modules.git//modules/code-server?ref=v1.0.19"
+     agent_id = coder_agent.example.id
+     offline  = true # Prevent external downloads
+   }
+   ```
```

Comment:
Make it more explicit, taking care of the new `registry/namespace/modules/module-name structure.